### ISSUE TITLE
Bitte den Docker Container neu Builden <3

### ISSUE THREAD

**Issue by Mega-Bits** [Pfuenzle/anime-loads#13]
Wie im Titel und danke für die großartige Arbeit

**Comment by Pfuenzle** [comment]
Stimmt, man muss jetzt leider bei DockerHub zahlen, um Container automatisch bei ändere zu bauen, muss mir was anderes überlegen.


Neues Image ist aber jetzt da ^^
